frontend/src/scenes/session-recordings/player/sidebar/PlayerSidebarOverviewTab.tsx

```diff
@@ -0,0 +1,93 @@
+import { Tooltip } from '@posthog/lemon-ui'
+import { useValues } from 'kea'
+import { PropertyIcon } from 'lib/components/PropertyIcon'
+import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
+import { getCoreFilterDefinition } from 'lib/taxonomy'
+import { ReactNode } from 'react'
+import { countryCodeToName } from 'scenes/insights/views/WorldMap'
+
+import { SessionRecordingType } from '~/types'
+
+import { playerMetaLogic } from '../playerMetaLogic'
+import { sessionRecordingPlayerLogic } from '../sessionRecordingPlayerLogic'
+
+interface OverviewItem {
+    label: string
+    value: ReactNode
+    tooltipTitle?: string
+}
+
+const browserPropertyKeys = ['$geoip_country_code', '$browser', '$device_type', '$os']
+const mobilePropertyKeys = ['$geoip_country_code', '$device_type', '$os_name']
+const recordingPropertyKeys = ['click_count', 'keypress_count', 'console_error_count'] as const
```

Comment:
```suggestion
const recordingPropertyKeys = ['click_count', 'keypress_count', 'console_error_count']
```

Reply:
This was to [help out Typescript here](https://github.com/PostHog/posthog/pull/25140/files#diff-edd0fe8805da0d91d82a3ada43d365dd2637b55bc89d4fceca8834d695215c27R160), otherwise it's complaining about string not being able to index to object.